### ISSUE TITLE
Setting up debugger in vs code

### ISSUE THREAD

**Issue by lavaxun** [glennreyes/graphpack#60]
Any documentation or guide for setting up debugger in visual studio code?

**Comment by gomesalexandre** [comment]
You'll need to either launch the `graphpack` CLI with node debug enabled from your app, or it could be added as a flag to the CLI, so you could have a debug script in your `package.json` such as:
```
{
  "dev:inspect": "graphpack --debug"
}
```

You'll then have to attach your VSC debugger to the running Node.js process 

This will be a quick win, I'll create a PR soon, hopefully @glennreyes sees benefit in it!

**Comment by glennreyes** [comment]
Yeah, I think that'd would work for me. A PR would be awesome @gomesalexandre! 🙌
